test/hotspot/jtreg/serviceability/dcmd/thread/VThreadCommandsTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2023, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Reply:
The test initially created in Oct 2023. It might look like a new test in main line but I think we are supposed to keep the original creation date.